src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -3272,17 +3290,28 @@ void AutomationInstrumentClipView::displayParameterName(int32_t paramID) {
 //display parameter value when it is changed
 void AutomationInstrumentClipView::displayParameterValue(int32_t knobPos) {
 
+	lastPadSelectedKnobPos = knobPos;
+
 	char buffer[5];
 
 	intToString(knobPos, buffer);
```

Comment:
if a pad is currently being held, then we need to use different drawing functions for the popup to prevent the popup from disappearing while we're still holding the pad

otherwise if not being held, use the regular displayPopup function.